napalm_ios/ios.py

```diff
@@ -670,15 +675,61 @@ def get_optics(self):
 
         return optics_detail
 
+    def get_cdp_neighbors(self):
```

Comment:
NAPALM requires a uniform interface across all drivers...so we would be (very) unlikely to have a custom Cisco only method call...so yes the call would be to `get_lldp_neighbors()`.

Longer term there should maybe be discussions to rename this or to have a secondary name like `get_layer2_neighbors`.

If they really want to they can call the private method.